webui/react/src/services/apiConfig.ts

```diff
@@ -435,6 +438,92 @@ export const getResourceAllocationAggregated: DetApi<
   },
 };
 
+/* Trials */
+export const queryTrials: DetApi<
+  Api.V1QueryTrialsRequest,
+  Api.V1QueryTrialsResponse,
+  Api.V1QueryTrialsResponse
+> = {
+  name: 'queryTrials',
+  postProcess: (response: Api.V1QueryTrialsResponse): Api.V1QueryTrialsResponse => {
+    return response;
+  },
+  request: (params: Api.V1QueryTrialsRequest) => {
+    return detApi.TrialsComparison.queryTrials({
+      ...params,
+      limit: params?.limit ? 3 * params.limit : 30,
```

Comment:
https://github.com/determined-ai/determined/blob/53ed8b324d37f76c8d184d3dfd14f041f15763b5/webui/react/src/pages/TrialsComparison/Table/TrialTable.tsx#L527-L541

this is in lieu of having a total for pagination (which would be costly for performance). it fetches the current page ) along with the next two pages- if they exist- and then we trim it down here `dataSource={trials.data.slice(0, settings.tableLimit)}`. 

so this is basically to know whether to know whether the next two pages exist so we know what `fakeTotal` to tell antd so that it will display the correct number of additional pages in the pagination control.

it's not pretty, and i think it would probably be better to implement backend side at some point but meh ¯\_(ツ)_/¯